content/realtime/usage.textile

```diff
@@ -155,6 +156,18 @@ blang[objc,swift].
 
   **Note: Since version 1.0.8, the iOS client libraries are thread-safe.** Check "the README":https://github.com/ably/ably-cocoa#thread-safety for details.
 
+blang[flutter].
+  The Ably library is "hosted on Github":https://github.com/ably/ably-flutter and is available as a "Dart package":https://pub.dev/packages/ably-flutter/. Update your @pubspec.yaml@ with appropriate package version:
```

Comment:
Please avoid using "Dart package" as that is misleading for non-Flutter developers. I would prefer "Flutter plugin" or "plugin" for clarity. e.g.:

```suggestion
  The Ably library is "hosted on Github":https://github.com/ably/ably-flutter and is available as a "Flutter plugin":https://pub.dev/packages/ably-flutter/. Update your @pubspec.yaml@ with appropriate package version:
```

I see there's at least one other instance in this PR where this will need fixing also.

Reply:
addressed 4770624984d162f1ca4416b94db7dd50f643e400